dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
+
+namespace Microsoft.ProjectReunion
+{
+    // Enums duplicated from IPowerManagerStatics
+    enum EnergySaverStatus
```

Comment:
Looks like we still need to do a review of the power-state stuff in @andreww-msft 's [branch](https://github.com/microsoft/ProjectReunion/compare/user/andreww-msft#diff-150d556fe5d95f35b9cbada87b6ad1a00bdd78f539634970cc6b38feedcd5a67). I'll prioritize that for the coming week.  This IDL needs some help before we merge it in.  Some initial things here you can get started on...